src/layouts/en/download.html.jade

```diff
@@ -0,0 +1,77 @@
+doctype html
+html(lang="fr")
+    head
+        meta(charset="UTF-8")
+        link(rel="shortcut icon", href="/assets/images/favicon.png")
+        title="Cozy Documentation"
+        meta(name="description", content="It is the place to find Apps from Cozy. Cozy brings is a personal cloud platform that allows you to store, sync and share data the way you want.")
```

Comment:
it is, we put that here because it's easy and fast to deploy, it'll end up on the website (currently ongoing work by the engagement team)